transform/mattermost-analytics/macros/rudderstack/get_rudderstack_columns.sql

```diff
@@ -0,0 +1,12 @@
+{% macro get_rudderstack_columns() %}
+
+   {%-
+        set rudderstack_columns = ['id', 'anonymous_id', 'received_at', 'sent_at', 'original_timestamp', 'timestamp', 
+        'context_ip', 'event', 'event_text']
+    -%}
+
+    {% for column in rudderstack_columns %}
+    , {{ column }} AS {{ column }}
+    {% endfor %}
```

Comment:
Following the example [here](https://docs.getdbt.com/docs/get-started/learning-more/using-jinja#use-looplast-to-avoid-trailing-commas), the code can be improved to avoid leading commas.
```suggestion
    {% for column in rudderstack_columns %}
        {{ column }} AS {{ column }}
        {% if not loop.last %},{% endif %}
    {% endfor %}
```